src/client/components/SliderComponent/SliderComponent.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import SliderComponent from './SliderComponent';
+
+export default { title: 'Slider Component' };
+
+export const Slider = () => <SliderComponent />;
```

Comment:
It does not seem to come up in my storybook. Is it currently working for you?

Reply:
Yes It is working. Can I ask any of other friends to try and check if its working for them?

Reply:
I asked one of my friends to try and they say its working for them.